### ISSUE TITLE
SVG y-axes

### ISSUE THREAD

**Issue by nycoliver** [skishore/makemeahanzi#10]
Hi thanks for the fantastic library!!

Just wondering why the svg y axes are inverted. It would be nice to render them without applying a transform each time.

Thanks

**Comment by skishore** [comment]
Hey, you raise an excellent point - there's no good reason for the y-axis to be inverted in the dataset.

I wrote a [tool](https://github.com/skishore/makemeahanzi/tree/tool) to generate this data, and within that tool the y-axis should be inverted, because that's the coordinate system that the TrueType font I derived these graphics from uses. However, there's a clearly delineated [export step](https://github.com/skishore/makemeahanzi/blob/tool/server/migration.js#L235) in this tool, and at that point, it would be easy to correct the coordinate system for consumers.

If you're willing to submit a pull request, I'd be happy to take a look! If not, I will probably get to this sometime myself.
